packages/utils/src/v2/TableId.ts

```diff
@@ -26,6 +27,10 @@ export class TableId {
     return arrayToHex(tableId);
   }
 
+  static fromHexString(tableId: string) {
+    return TableId.fromBytes32(hexToArray(tableId));
+  }
+
   static fromBytes32(tableId: Uint8Array) {
     // assumes tableId is a 32-byte hex string, otherwise it left-pads with zeros (for numbers)
     // this is scary, since zero padding is different depending on the type (bytes types vs number types)
```

Comment:
A comment about line 39 (can't comment on there bc it wasn't changed in this PR): the `tableId.reverse()` reverses the array in place, this tripped me up today when using the `fromBytes32` method as a util to print a table id I was debugging. Could we first copy it to a fresh array and do the `reverse` there to avoid that?

Reply:
oh yes, definitely! I also learned today that TypedArrays are mutable and Uint8Array, etc. can be "views" into larger arrays under the hood 🙈 which caused ~half of this refactor